docs/_documentations/remote-configuring.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: docs
+title: Configuring Codewind remote
+description: Configuring your environment to use Codewind remote
+keywords: users, projects, Kubernetes, LDAP, user management, access management, login, deployment, pod, security, securing Cloud connection
+duration: 5 minutes
+permalink: remoteconfiguring
+type: document
+parent: installoncloud
+order: 1
+---
+
+## Configuring Codewind remote
+
+To securely configure Codewind remote, do the following steps.
+
+1. Optional: Create a namespace to install into by running the following command: `kubectl create namespace <your_namespace>`. If you do not create a new namespace, Codewind uses the current one.
+2. If you are developing on macOS, you must install an ingress load balancer into your Kube environment. For example: 
```

Comment:
We need to change this line. We should tell users to do that **only** if they're installing on Docker Desktop Kubernetes, or don't have an ingress controller installed on their cluster.

Reply:
Redrafted, but will require another review, and there is still some info missing. Closing this as resolved in lieu of another review and obtaining said info.